client-js/app/elements/labrad-registry.ts

```diff
@@ -366,11 +377,11 @@ export class LabradRegistry extends polymer.Base {
     var newPath = JSON.parse(this.$.copyPathInput.value);
 
     try {
-      if (this.selectType === 'dir') {
-        await this.socket.copyDir({path: this.path, dir: this.selDir, newPath: newPath, newDir: newName});
+      if (this.selectedType == 'dir') {
```

Comment:
Still best to use triple equals here (and numerous places below). Double equals can do weird type coercions and it's best not to get in the habit of using it.